core/components/_helpers/globals.js

```diff
@@ -113,5 +87,15 @@ if (includeGlobals) {
       -webkit-font-smoothing: antialiased;
       -moz-osx-font-smoothing: grayscale;
     }
-  `)
-}
+`
+
+const includeGlobals = process && process.env && process.env.COSMOS_DISABLE_RESETS
```

Comment:
Hi 👋

There's a flag for this behavior already: COSMOS_DISABLE_RESETS

Docs: auth0-cosmos.now.sh/docs/#/usage#usage-with-styleguide

Unless I'm missing something here, we can re-use the same flag

Reply:
hi! 🤔 Are you referring to the new flag I added? I also thought of reusing but that would be a breaking change if I'm not wrong (should have left a comment about that, sorry).
My reasoning:
- COSMOS_DISABLE_RESETS=true => minimum style resets
- COSMOS_DISABLE_RESETS=false => recommended style resets
- COSMOS_MANUAL_RESETS=true => none of the resets applied (to avoid knowing how to do it and for example depending on `document`)
- COSMOS_MANUAL_RESETS=false => include the resets driven by the COSMOS_DISABLE_RESETS variable

I think the COSMOS_DISABLE_RESETS behavior should be more aligned with it's name and have the effect that the MANUAL one has, but I didn't want to break the interface we have in the current version.

If my reasoning is right, feel free to take a decision over breaking or not and I can apply it 🙂